allensdk/brain_observatory/ecephys/_channels.py

```diff
@@ -0,0 +1,107 @@
+from typing import List, Optional
+
+import numpy as np
+import pandas as pd
+from pynwb import NWBFile
+
+from allensdk.brain_observatory.ecephys._channel import Channel
+from allensdk.brain_observatory.ecephys.utils import clobbering_merge
+from allensdk.core import DataObject, NwbReadableInterface, \
+    JsonReadableInterface
+
+
+class Channels(DataObject, NwbReadableInterface, JsonReadableInterface):
```

Comment:
I think it is unnecessary to create Channels and Probes data classes. These are just lists of Channel and Probe. Instead we should try reusing available methods for adding multiple probes like `add_probewise_data_to_nwbfile(nwbfile, probes)` or make our own methods if available somehow do not generalize to our tasks.

Reply:
I disagree. I don't like passing a dictionary around and want to pass around strongly typed data.